libminifi/include/core/state/Value.h

```diff
@@ -228,36 +225,36 @@ class IntValue : public Value {
   }
 
  protected:
-  virtual bool getValue(int &ref) {
+  bool getValue(int &ref) override {
     ref = value;
     return true;
   }
 
-  virtual bool getValue(uint32_t &ref) {
+  bool getValue(uint32_t &ref) override {
     if (value < 0)
       return false;
     ref = value;
     return true;
   }
 
-  virtual bool getValue(int64_t &ref) {
+  bool getValue(int64_t &ref) override {
     ref = value;
     return true;
   }
 
-  virtual bool getValue(uint64_t &ref) {
+  bool getValue(uint64_t &ref) override {
     if (value < 0)
```

Comment:
should we use the new `utils::internal::cast_if_in_range` here as well?

Reply:
We could, I left these signed into longer unsigned as is because they could be confidently casted with one if.
Of course making everything the same could make this more readable. Should I change it?

Reply:
I say we go for it if the refactor is not substantial

Reply:
refactored it in [4b341dc](https://github.com/apache/nifi-minifi-cpp/pull/1032/commits/4b341dccf0c75efb76968acb467e6af5996f51d6)